### DISCUSSION TITLE
Project broke after migration to ItwinUI V3

### DISCUSSION THREAD

**Discussion by gustavo-conti** [iTwin/iTwinUI#2354] [Q&A] [answered]
We migrated from version 2 to 3 of Itwin UI, all unit tests are passing, but when trying to run the project the error below is displayed
![image](https://github.com/user-attachments/assets/f8c1003c-4d78-484c-a29f-dbebac564d65)
![image](https://github.com/user-attachments/assets/1931232e-f5f7-452d-932b-af44a8c99a84)

Versions

@itwin/itwinui-react: "^3.15.5
 react: "^17.0.2",
 react-dom: "^17.0.2",

**Comment by r100-stack** [comment, marked as answer]
Thanks for migrating to v3 🙂

I believe you are somehow missing the `value` prop that is passed to each `Tabs.Tab` when using the [Composition API](https://itwinui.bentley.com/docs/tabs#composition-api). Could you please try creating a [repro](https://stackblitz.com/github/iTwin/iTwinUI/tree/main/minimal-sandbox) after following the examples in [Tab docs](https://itwinui.bentley.com/docs/tabs)?
